var/spack/repos/builtin/packages/adios2/package.py

```diff
@@ -24,14 +24,21 @@ class Adios2(CMakePackage):
 
     variant('shared', default=True,
             description='Also build shared libraries')
+    variant('pic', default=True,
+            description="Enable position independent code "
+                        "(for usage of static in shared downstream deps)")
```

Comment:
If there's going to be a `pic` variant then it should be mutually exclusive with `shared` since `pic` is really "static + PIC"

Reply:
yep, unfortunately we cannot hide this variant on `~shared` only. therefor, we will cover this in the logic below.